examples/flutter_gallery/lib/demo/animation/sections.dart

```diff
@@ -43,7 +43,7 @@ class Section {
   final List<SectionDetail> details;
 
   @override
-  bool operator==(Object other) {
+  bool operator ==(Object other) {
```

Comment:
this is good.
i wish the next line would get linted, though. or generally caught.